src/controllers/signIn.js

```diff
@@ -0,0 +1,5 @@
+exports.get = (req,res) => {
+    res.render("sign-in", {
+        title: "Sign-in", headerFound: false, footerFound: false, asideFound:false, style: ["signUpIn"]
```

Comment:
require sign in , sign up js files

Reply:
yeah ons , link the validation files to hbs !!!